lib/moment/juttle-moment.js

```diff
@@ -844,6 +788,66 @@ var JuttleMoment = Base.extend({
         }
 
         return operand.valueOf();
+    },
+    momentString: function(moment) {
+        if (isNaN(moment.valueOf())) {
+            return moment._i.toString();
+        } else {
+            return moment.toDate().toISOString();
+        }
+    },
+    simpleDurationString: function(d) {
```

Comment:
PROD-8585, there is also a PR by @welch in our old repo referenced from there (could be used as a base but it has a couple of un-addressed comments).